regexes/client/browsers.yml

```diff
@@ -81,7 +86,7 @@
     default: 'Gecko'
 
 # SeaMonkey
-- regex: '(Iceape|SeaMonkey|gnuzilla)(?:/(\d+[\.\d]+))?'
+- regex: '(Iceape|IceCat|SeaMonkey|gnuzilla|Mobicip)(?:/(\d+[\.\d]+))?'
```

Comment:
For v4 I will no longer allow dynamic names for browser detections. Would be better to split them up and have one regex each. That will also guarantee that the name has always the same case...

Reply:
There is actaully no test for `gnuzilla`, and I dont have a useragent for this browser. Do you have one?

Reply:
We could use this one: https://developers.whatismybrowser.com/useragents/parse/210967-linux-gecko

But actually I'm not sure if "browser" would be the correct classification for GNUZilla, as it actually was a application suite containing multiple tools like browser, email, .... Maybe we should move it to pim, or remove it completely 🤔 
Same actually applies to Iceape and SeaMonkey. All of them were modified versions of the `Mozilla Application Suite`